### PR TITLE
Added new page to view tasks

### PR DESCRIPTION
**Pull request by Meghana359**
1. Updated get_create_form to display a form.
2. Added a new page to view all the tasks. [/viewTask will fire up the page]
3. Deleted dummy Model which I created in the beginning.

### CODE REVIEW

File: task_manager/api.py [hunk 1]
```diff
@@ -46,6 +46,12 @@ def get(self):
         return auth.logout()
 
 
+@api.route('/viewTask')
```
Comment (SemicolonExpected) [inline task_manager/api.py:49]: could we route this to just /tasks? I think that might be an easier URL to remember
and then individual tasks be /tasks/{taskid}

**Review by SemicolonExpected — CHANGES_REQUESTED** [review]

File: task_manager/api.py (hunk 1 above)
Comment (Meghana359) [reply task_manager/api.py:49]: currently, /task gives JSON response. I couldn't combine both JSON and rendering template in a way where JSON could be accessed outside the template.

Comment (SemicolonExpected) [reply task_manager/api.py:49]: maybe it should be specifically /createTask then that returns the json?

I'm looking at `@task_ns.route('/')` which is /tasks I'm assuming and this seems to be fetching tasks in general, so its what the "get" is suppose to do. What we can probably do is just change the route from /viewTask to `@task_ns.route('/')`  and then remove its definition underneath?

**Review by SemicolonExpected — CHANGES_REQUESTED** [review]

File: task_manager/api.py (hunk 1 above)
Comment (Meghana359) [reply task_manager/api.py:49]: Yeah, I can change it to @task_ns.route('/'viewTask). It makes sense. But @task_ns.route('/') routes to /tasks that returns all the tasks in JSON.

Comment (SemicolonExpected) [reply task_manager/api.py:49]: Oh so is this just the function to get the task while the page /tasks/{taskid} will show it?

**Pull request merged**